graphkb/util.py

```diff
@@ -88,3 +81,189 @@ def convert_aa_3to1(three_letter_notation: str) -> str:
 
     result.append(three_letter_notation[last_match_end:])
     return ''.join(result)
+
+
+def join_url(base_url: str, *parts) -> str:
+    """
+    Join parts of a URL into a full URL
+    """
```

Comment:
On lines 57 and 64, docstrings has been transposed into one-liners. Which way should be prefered?

Reply:
I think the single line is the suggestions of pydocstyle, so I just go with it's suggestions.